Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -464,6 +464,8 @@ createDriverDetails personId adminId merchantId = do
             canDowngradeToTaxi = False,
             mode = Just DriverInfo.OFFLINE,
             lastEnabledOn = Just now,
+            blockedReason = Nothing,
```

Comment:
Where are you storing block code?

Reply:
it will be redundant , because we are storing reason also .

Reply:
Then what is the point of having blockCode?